File: Aula2/Eq2/Eq2.cpp
#include <iostream>
#include <iomanip>
#include <cmath>

int main()
{
    float a, b, c;
    
    std::cin >> a >> b >> c;
    
    float delta = (float) sqrt((b*b) - (4.00*a*c));
    float primeira_raiz = (float) (-b + delta)/(2.00*a);
    float segunda_raiz = (float) (-b - delta)/(2.00*a);
    
    std::cout << std::fixed;
    std::cout << std::setprecision(4) << primeira_raiz << " " << std::setprecision(4) << segunda_raiz << std::endl;
    return 0;    
}

File: Aula1/Eq2/Eq2.java
import java.util.Scanner;
import java.lang.Math;

public class Eq2{
    public static void main (String[] args) {
        double a, b, c;
        Scanner sc = new Scanner(System.in);
        a = sc.nextDouble();
        b = sc.nextDouble();
        c = sc.nextDouble();
        
        double delta = Math.sqrt((Math.pow(b, 2.00) - (4*a*c)));
        double primeiraRaiz = (-b + delta)/(2*a);
        double segundaRaiz = (-b - delta)/(2*a);
        
        String str1 = String.format("%.4f", primeiraRaiz);
        String str2 = String.format("%.4f", segundaRaiz);
        System.out.println(str1 + " " + str2);
        
    }
}